controllers/directories.js

```diff
@@ -1,11 +1,12 @@
 const fs = require('fs');
 const path = require('path');
 const config = require('config');
+const lccGetter = require('../utils/lcc-body-getter');
 
 function _get (req, res, next) {
+  const lcc = lccGetter(req)
   const relativePath = req.query.path || '';
-  const absolutePath = path.resolve(config.get('baseDir'), req.user.username, relativePath);
-
+  const absolutePath = path.resolve(config.get('baseDir') + lcc, req.user.username, relativePath);
```

Comment:
```suggestion
  const absolutePath = path.resolve(config.get('baseDir'), lcc, req.user.username, relativePath);
```
path.resolve will concatenate automatically. Remember to remove the slash in lcc-body-getter (e.g. '/lcc1' to 'lcc1')